### DISCUSSION TITLE
Open Shell Taskbar button question

### DISCUSSION THREAD

**Discussion by dmcd1950** [Open-Shell/Open-Shell-Menu#1710] [Q&A]
Is there a way to enable Never Combine Taskbar Buttons in Win 11 using Open Shell?

**Comment by bonzibudd** [comment]
This is a feature in an upcoming release of Windows 11, so Open-Shell doesn't need to do anything about it. There is also ExplorerPatcher.

**Comment by dmcd1950** [comment]
Thank you very much for your quick reply.  It is good to know that this will be addressed sometime in the future with a new release of Win 11, but that doesn't help us now.  When I did a search for a fix for this issue I found that many  others are looking for a fix also.  This problem is cutting down efficiency and costing time and frustration.  I'm sure that is why it is being addressed by MS.
I was told that Open Shell included a fix, that is why I installed it.
Thanks for your help.  I will see if I can fix  it with Explorer Patch.
Bill
